subsys/bluetooth/controller/ll_sw/ull.c

```diff
@@ -223,13 +223,12 @@
 /* Note: Need node for PDU and CTE sample */
 #define BT_CTLR_ADV_EXT_RX_CNT  (CONFIG_BT_CTLR_SCAN_AUX_SET * \
 				 CONFIG_BT_CTLR_DF_PER_SCAN_CTE_NUM_MAX * 2)
-#else
-/* Note: Assume up to 7 PDUs per advertising train (max data length) */
-#define BT_CTLR_ADV_EXT_RX_CNT  (CONFIG_BT_CTLR_SCAN_AUX_SET * 7)
-#endif
-#else
+#else /* !CONFIG_BT_CTLR_DF_CTE_RX */
+#define BT_CTLR_ADV_EXT_RX_CNT  1
```

Comment:
For CTE the `BT_CTLR_ADV_EXT_RX_CNT` is still controlled by `CONFIG_BT_CTLR_SCAN_AUX_SET`. Is that intentional?

Reply:
@Thalley I did not change the DF related allocation. Otherwise, the background is that `PDU_RX_CNT` is the minimum allocation summation such that we reserve 3 (1 for adv data, 1 for scan response, 1 for empty PDU reception), 1 for minimum aux PDU reception (when advertising extensions is supported) and `CONFIG_BT_CTLR_RX_BUFFERS` that is shared across states/roles (including mul;tiple simultaneous chain PDUs reception).

@ppryga Could you answer if `BT_CTLR_ADV_EXT_RX_CNT` can be `1`  for DF too? The reason is, the `PDU_RX_CNT` is shared across states/roles and allocating here does not guarantee availability for IQ samples when other state/roles use the free node rx.

Reply:
@ppryga Any input?

Reply:
Sorry for late answer. The number of allocated PDUs in regard of DF is related with periodic chain length. It has to be controllable by end user. It depends on end user how many CTEs need to be send in a single periodic advertising event.
The BT Core spec allows for up to 16 CTEs in a chain.

Reply:
@cvinayak Does this require any change to this PR then?

Reply:
No change required in this PR, I need to rebased the PR though :-(